### ISSUE TITLE
unlock with TouchID, macOS 

### ISSUE THREAD

**Issue by phenomeus** [portfolio-performance/portfolio#4084]
**Is your feature request related to a problem? Please describe.**
because there is a portfolio performance client for iOS, its a good idea to lock the portfolio with a password and keep it on iCloud for syncing purposes. Unlocking with FaceID on iOS is super easy but now I need to always enter the Password on macOS Desktop.  

**Describe the solution you'd like**
It would be awesome to be able to use TouchID to unlock the macOS desktop client 

hope this is something that can be considered as a UX bump for portfolio performance

**Comment by PaolinPaperin** [comment]
I agree, that would be so convenient. Hope they decide to implement it, as I love using it on the iOS app.

**Comment by FlipEverything** [comment]
+1 for this feature. I’d also really appreciate Touch ID support on macOS - it would make unlocking encrypted portfolios much more convenient while keeping the current security benefits.
